### ISSUE TITLE
Adding Remember Me Setting?

### ISSUE THREAD

**Issue by weekender23** [barbequesauce/Watcher3#29]
hi

Everyday i have type in my login and password.  Is there way to add "Remember Me", like what Sonarr has?

**Comment by barbequesauce** [comment]
Why not just disable the WebUI password?

**Comment by weekender23** [comment]
If i did that, could someone (thats not me) change the settings in Watcher?

**Comment by barbequesauce** [comment]
That's what the option under it in the settings does - password-protects the settings..

**Comment by ellnic** [comment]
I think what he wants is to leave the password on, have the UI facing the outside and then be able to access and login to it on the run. But: have it remember him with a cookie like Sickbeard.

**Comment by barbequesauce** [comment]
Yeah I wouldn’t do that... CherryPy doesn’t currently have any unpatched vulnerabilities, and this code base seems solid - but I can’t vouch for its security in a public-facing scenario..

**Comment by barbequesauce** [comment]
Closing due to inactivity

**Comment by jandreaucodes** [comment]
Can I bump this to reopen it or should I submit a new request?

I have Watcher public through an nginx reverse proxy and would love to leave authentication on (in case someone discovers my super-secret URL) but not have to login so frequently.

**Comment by barbequesauce** [comment]
Parking research. https://groups.google.com/forum/m/#!topic/cherrypy-users/3rggaNVNjKE
